SU-SWS/stanford_profile
# D8CORE-6058 Adjust, improve, and add metatags for content

# READY FOR REVIEW

# Summary
- Add and improve automatic metatags for nodes
- Added ability for admins to customize metatags per page.

# Need Review By (Date)
- 9/9

# Urgency
- medium

# Steps to Test
1. `composer require su-sws/stanford_profile:dev-D8CORE-6058`
2. install site or import configs `drush cim -y`
3. create a basic page and experiment with the metatags
4. verify they are represented in the source code after saving.
